### DISCUSSION TITLE
Intel RVP board "RPL-S UD5-1DPC 4L micro ATX CRB FAB1

### DISCUSSION THREAD

**Discussion by leenaarora18** [zephyrproject-rtos/zephyr#80035] [Q&A]
We have an intel rpl board with EC mec1521 on it. Now we want to flash the image (bios +bin) onto it using dediprogrammer. We have intel bios which we can stitch with bin files using the fit tool. My question is while building it on Zephyr do I use the command "west build -c -p auto -b mec1501_mtl_p" with a MEC 1521x spi image.

**Comment by nordicjm** [comment]
@edersondisouza

**Comment by nashif** [comment]
@albertofloyd FYI
